frame/oracle/src/lib.rs

```diff
@@ -962,6 +1092,47 @@ pub mod pallet {
         None => Ok(()),
       })
     }
+
+    fn set_initial_market_pairs() {
+      if Self::supported_market_pairs().is_empty() {
+        let initial_supported_market_pairs: BoundedVec<MarketPair, T::SupportedMarketPairsLimit> =
+          BoundedVec::try_from(vec![
+            // ATH_USDC
+            MarketPair {
+              base_asset: Asset::AllTimeHigh.currency_id(),
+              quote_asset: Asset::USDCoin.currency_id(),
+            },
+            // BTC_USDC
+            MarketPair {
+              base_asset: Asset::Bitcoin.currency_id(),
+              quote_asset: Asset::USDCoin.currency_id(),
+            },
+            // ETH_USDC
+            MarketPair {
+              base_asset: Asset::Ethereum.currency_id(),
+              quote_asset: Asset::USDCoin.currency_id(),
+            },
+            // TDFY_BTC
+            MarketPair {
+              base_asset: Asset::Tdfy.currency_id(),
+              quote_asset: Asset::Bitcoin.currency_id(),
+            },
+            // TDFY_ETH
+            MarketPair {
+              base_asset: Asset::Tdfy.currency_id(),
+              quote_asset: Asset::Ethereum.currency_id(),
+            },
+            // TDFY_USDC
+            MarketPair {
+              base_asset: Asset::Tdfy.currency_id(),
+              quote_asset: Asset::USDCoin.currency_id(),
+            },
+          ])
```

Comment:
`ETH_BTC` should be included in the initial pairs.

Reply:
ditto